develop/dev-guide-read-in-fast-mode.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Fast Mode
+summary: Introduces a way to speed up the querying of AP scenarios by using Fast Mode.
+---
+
+# Fast Mode
+
+This document describes how to use Fast Mode to speed up queries in Analytical Processing (AP) scenarios.
+
+TiFlash supports the following modes:
+
+- `Normal Mode` mode. The default mode. This mode guarantees the accuracy of query results and data consistency.
+- `Fast Mode` mode. This mode does not guarantee the accuracy of query results and data consistency, but provides more efficient query performance.
+
+AP allows for some tolerance to the accuracy of the query results. If you need higher query performance, you can switch the corresponding table to TiFlash's Fast Mode for querying.
+
+The mode switch takes effect globally. TiFlash mode switching is only available for tables with TiFlash Replica. TiFlash-related operations are not supported for temporary tables, in-memory tables, system tables, and tables with non-utf-8 characters in the column names, and therefore changing TiFlash table mode is not supported for them.
+
+For more information, see [ALTER TABLE SET TIFLASH MODE](/sql-statements/sql-statement-set-tiflash-mode.md).
+
+## Switch to Fast Mode
+
+By default, all tables are in Normal Mode. You can use the following statement to view the current table mode.
+
+```sql
+SELECT table_mode FROM information_schema.tiflash_replica WHERE table_name = 'table_name' AND table_schema = 'database_name'
+```
+
+Use the following statement to switch the corresponding table to Fast Mode.
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE table_name SET TIFALSH MODE FAST
+```
+
+Once the switch is complete, subsequent queries in TiFlash will be performed in Fast Mode.
+
+You can switch back to Normal Mode using the following statement.
+
+```sql
+ALTER TABLE table_name SET TIFALSH MODE NORMAL
+```
+
+## Mechanism of Fast Mode
+
+The data in the stroage layer of TiFlash is stored in two layers: Delta layer and Stable layer.
```

Comment:
stroage --> storage
I think we may can change the expression of 'storage layer'. Too many 'layers' may make a little weird.

Reply:
fixed